xml/klp.xml

```diff
@@ -367,9 +367,9 @@ Activate with: SUSEConnect -p sle-module-live-patching/&productnumber-regurl;/x8
       patches</command> command. You can see the currently running patch on the
       line starting with <literal>RPM:</literal>. For example:
      </para>
-<screen>RPM: kernel-livepatch-5_3_18-24_29-default-2-2.1.x86_64</screen>
+<screen>RPM: kernel-livepatch-6_4_0-150600_9-default-1-150600.2.36.x86_64</screen>
```

Comment:
why not entities here?

Reply:
Because for the kernel patches, the version number is spelled with underscores and hyphens instead of dots (unlike the RPM version) and it also includes more numbers after the `-default` part of the string. I did not spot this very format of the version number anywhere else in the docs that's why I thought it would not make sense to create an entity for it (plus I did not see a way to deduct this from the RMP package name in any way - I had to run the command to see it.

Reply:
understood, reasonable :-)